### DISCUSSION TITLE
How can the default node version be set using NVM?

### DISCUSSION THREAD

**Discussion by SuperJUNOed** [community/community#50658] [Actions] [answered]
### Select Topic Area

Question

### Body

I have installed nvm (ubuntu with zsh shell) with two node version: v6.11.5 and v9.0.0 and the default version in nvm is the v9.0.0

Every time I need to change the node version

$ nvm list
         v6.11.5
->       v9.0.0
         system
default -> node (-> v9.0.0)
node -> stable (-> v9.0.0) (default)
stable -> 9.0 (-> v9.0.0) (default)

$ nvm v6
How could I change the nvm version default to define v6.11.5?

**Comment by smartman0307** [comment, marked as answer]
nvm alias default 6.11.5 if you want it pegged to that specific version.

You can also do nvm alias default 16 or nvm alias default node.

Either way, you'll want to upgrade to the latest version of nvm (v0.39.2 as of this writing)

# nvm set default node.js version 16.14.2
$ nvm alias default 16.14.2
$ nvm use

$ node -v
# v16.14.2
